doc/contributing/internals/caching.rst

```diff
@@ -0,0 +1,25 @@
+Caching
+=======
+
+The caching strategies are stored in
+``remoteStorage.caching._rootPaths``. For instance, on
+https://myfavoritedrinks.remotestorage.io/, it has the value ``{
+/myfavoritedrinks/: "ALL" }``.
+
+These rootPaths are not stored in localStorage. If you refresh the page,
+it is up to the app to set all caching strategies again during the
+page load.
+
+The effect of the caching strategy is basically achieved through three
+paths:
+
+1. Setting caching strategy 'ALL' for a path creates an empty node for
+   that path, unless it already exists.
+2. The sync process will then do a 'GET', and create new nodes under any
```

Comment:
I think this should be "The sync process will then do a GET request", at first I thought it was the same as 'ALL' above because of the quotes